public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -39,13 +40,28 @@ export function getFillColor(cfg: StyleConfigValues) {
   return undefined;
 }
 
+const textLabel = (cfg: StyleConfigValues) => {
+  const fontFamily = config.theme2.typography.fontFamily;
+  const fontSize = cfg.textConfig?.fontSize ?? defaultStyleConfig.textConfig.fontSize;
```

Comment:
```suggestion
  const textConfig = { 
    ...defaultStyleConfig.textConfig,
    ...cfg.textConfig,
   }
```
then you won't need all the optional chaining below